src/components/activity-page/embeddable.scss

```diff
@@ -7,7 +7,7 @@
     width: 100%;
   }
   &.reduced-width {
-    width: 45%;
+    width: 440px;
```

Comment:
Old LARA uses 45% for the sizing of the non-full-width questions, but it results in layouts where content edges don't line up.  I think this results in a better display of the question content.